modules/op-configuring-custom-sa-and-scc.adoc

```diff
@@ -29,7 +31,7 @@ allowHostIPC: false
 allowHostNetwork: false
 allowHostPID: false
 allowHostPorts: false
-allowPrivilegeEscalation: false
+allowPrivilegeEscalation: true
```

Comment:
i would love to explain this further this under in the <1> <2> <3> ... section

https://github.com/openshift/openshift-docs/pull/61617/files#r1245188027